### DISCUSSION TITLE
JSON not recognizing a string item from an iteration

### DISCUSSION THREAD

**Discussion by kamui-moritaalbright** [langgenius/dify#9836] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [x] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I'm using a HTTP node inside an iteration. The input to the iteration is an array of strings, and I can see that the http node recognizes the string item extacted from the array when I place it in any of the following: url, headers, params. However, when I try to place the variable inside JASON, it's not recognized.

I'd very much appreciate it if anybody could tell me why this is happening.

**Comment by bowenliang123** [comment]
#12759
#12761
